### ISSUE TITLE
trouble logging in since mojave

### ISSUE THREAD

**Issue by steveoh** [stuartcryan/lastpass-alfred-workflow#52]
has anyone noticed anything odd since updating their os?

lplogin shows popup but it sometimes takes 30 seconds
after logging in it does not seem to ever authenticate

**Comment by carceneaux** [comment]
At some point, I did have to clear my Alfred cache after upgrading to Mojave but I haven't seen issues since.

https://www.alfredapp.com/help/advanced/#rebuild

**Comment by cadnza** [comment]
Same thing here; reloading the cache didn't fix it. Here's what the debugger says:
```
2018-10-10 15:57:49][ERROR: action.script] 2482:2784: execution error: Error: Unable to retrieve password from askpass (no reply) (1)
```

**Comment by steveoh** [comment]
and another error message after typing in password

`[ERROR: action.script] 2465:2767: execution error: Error: Couldn't resolve host name. (1)`

try again

`[ERROR: action.script] 2465:2767: execution error: /bin/bash: line 1:  3603 Segmentation fault: 11  /usr/local/bin/lpass login --trust "my@email.address" (139)`

looks like an issue in [lastpass-cli](https://github.com/lastpass/lastpass-cli). 

I `brew reinstall lastpass-cli --with-pinentry` and ran the `lpass login --trust` command and that seemed to make the plugin work.

**Comment by carceneaux** [comment]
LOL, well the scripts are open source so you see what's going down. I'd hope that you would've read those scripts prior to using an open-source utility to access your LassPass account.

Here are some ideas:

* After Mojave, there were some new security permissions I had to grant Alfred.
* The scripts are simply leveraging LastPass CLI. Have you tried taking Alfred out of the equation to see if LastPass CLI is working properly? - https://github.com/lastpass/lastpass-cli

**Comment by carceneaux** [comment]
Since my last post, I've rebooted my mac a number of times. I've noticed that following a reboot, I have to perform the following procedure to be able to log back in to LastPass:

1. Sync LastPass Vault (lpsync)
2. LastPass Login (lplogin)

For whatever reason, if I don't sync the LastPass Vault prior to logging in, I'm never logged in. Even with using the correct credentials.

Hope this helps someone!
